<launch>
    <param name="name" value="yoon"/>
    <node pkg="tcps_image_attack" type="node_airsim.py" name="airsim_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_yolo.py" name="yolo_node" output="screen">
    </node>
    <node pkg="tcps_image_attack" type="node_controller.py" name="controller_node" output="screen">
    </node>
</launch>
